godtools/Views/TractElements/TractButton.swift

```diff
@@ -60,16 +64,28 @@ class TractButton: BaseTractElement {
     override func setupView(properties: [String: Any]) {
         loadStyles()
         loadElementProperties(properties: properties)
-        self.height = self.properties.height
         
         self.button = GTButton()
-        button.cornerRadius = self.properties.cornerRadius
-        button.backgroundColor = self.properties.backgroundColor
+        if BaseTractElement.isModalElement(self) {
```

Comment:
Could you extract the if and else blocks out into separate functions, named something like `configureAsModalButton` and `configureAsStandardButton` or something like that?

I think that will help the code stay easily readable.